templates/carte/index.html.twig

```diff
@@ -0,0 +1,67 @@
+{% extends "static/layout.html.twig" %}
```

Comment:
index.html.twig is not a properly chosen name.
This template must not extend static / layout, you must create a /layout.html.twig which extends /base.html.twig in your map / folder